deepface/detectors/Yolov8faceWrapper.py

```diff
@@ -0,0 +1,71 @@
+from deepface.detectors import FaceDetector
+
+# Models names and paths
+PATHS = {
+    "yolov8n": "/.deepface/weights/yolov8n-face.pt",
+}
+
+# Google Drive base URL
+BASE_URL = "https://drive.google.com/uc?id="
+
+# Models' Google Drive IDs
+IDS = {
+    "yolov8n": "1qcr9DbgsX3ryrz2uU8w4Xm3cOrRywXqb",
+}
+
+
+def build_model(model: str):
+    """Function factory for YOLO models"""
+    from deepface.commons.functions import get_deepface_home
+
+    # Get model's weights path and Google Drive URL
+    func_weights_path = f"{get_deepface_home()}{PATHS[model]}"
+    func_url = f"{BASE_URL}{IDS[model]}"
+
+    # Define function to build the model
+    def _build_model(weights_path: str = func_weights_path, url: str = func_url):
+        import gdown
+        import os
+
+        # Import the Ultralytics YOLO model
+        from ultralytics import YOLO
+
+        # Download the model's weights if they don't exist
+        if not os.path.isfile(weights_path):
+            gdown.download(url, weights_path, quiet=False)
+            print(f"Downloaded YOLO model {os.path.basename(PATHS[model])}")
+
+        # Return face_detector
+        return YOLO(weights_path)
+
+    return _build_model
+
+
+def detect_face(face_detector, img, align=False):
+    resp = []
+
+    # Detect faces
+    results = face_detector.predict(
+        img, verbose=False, show=False, conf=0.25)[0]
+
+    # For each face, extract the bounding box, the landmarks and confidence
+    for result in results:
+        # Extract the bounding box and the confidence
+        x, y, w, h = result.boxes.xywh.tolist()[0]
+        confidence = result.boxes.conf.tolist()[0]
+
+        x, y, w, h = int(x - w / 2), int(y - h / 2), int(w), int(h)
+        detected_face = img[y: y + h, x: x + w].copy()
+
+        if align:
+            # Extract landmarks
+            left_eye, right_eye, _, _, _ = result.keypoints.tolist()
+            # Check the landmarks confidence before alignment
+            if left_eye[2] > 0.5 and right_eye[2] > 0.5:
```

Comment:
confidence score 0.5 should be defined with a variable in the module level.

Reply:
Like in the YoloWrapper?

```python
LANDMARKS_CONFIDENCE_THRESHOLD = 0.5
```

Reply:
yes this seems good

Reply:
ok